kuma/settings/common.py

```diff
@@ -1591,3 +1589,4 @@ def get_user_url(user):
 ce_path = path('kuma', 'settings', 'content_experiments.json')
 with open(ce_path, 'r') as ce_file:
     CONTENT_EXPERIMENTS = json.load(ce_file)
+`
```

Comment:
And remove this one as well.

Or, triple quotes if you mean to comment out the whole file 😁